index.html

```diff
@@ -6,6 +6,8 @@
   <title>Welcome to Real Dev Squad</title>
   <link rel="stylesheet" href="./css/common/main.css">
  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+ <link rel="icon" type="image/x-icon" href="img/logos/favicon.ico"/>
+ <meta name="theme-color" content="#e30062">
```

Comment:
This doesn't work for PC, only for android ?

Reply:
<meta name="theme-color" content="#e30062"> will work on mobile devices only